models/event.js

```diff
@@ -32,6 +33,9 @@ const MODEL = {
         .string().hex().length(40)
         .description('SHA this project was built on')
         .example('ccc49349d3cffbd12ea9e3d41521480b4aa5de5f'),
+    startFrom: requiresValue
```

Comment:
Calling out job name will be problematic, now it's again up to the caller to infer. Can we call it `trigger` ? 
`trigger` can be `~commit` / `~pr`. This will just trigger all jobs which `requires` on `~commit` / `~pr`

Also `trigger` can be `job:jobName` (or something) if we want to support starting pipeline at a specific job.

Reply:
Not sure what you mean by "Calling out job name will be problematic"? 
requiresValue can take in both `jobName` or `~commit`

Reply:
Scratch my comment, I misunderstood it as `startFrom` should be a `jobName`

Reply:
I'm not following the logic for why `trigger` is different from `startFrom` other than one has `job:name` syntax and the other does not.